ts/store/reducers/authentication.ts

```diff
@@ -35,7 +35,10 @@ import { GlobalState } from "./types";
 type LoggedOutReason = "NOT_LOGGED_IN" | "SESSION_EXPIRED";
 
 // PublicSession attributes
-export type TokenName = keyof Omit<PublicSession, "spidLevel">;
+export type TokenName = keyof Omit<
```

Comment:
Because `lollipopAssertionRef` is not a token and does not represent just a string. There is no need to retrieve it as a token from a function like `tokenFromNameSelector` and leaving it in type `tokenName` would be semantically incorrect. Also, it would throw an error in `tokenFromNameSelector` because its `createSelector` only expects `Option<string>` as it should for a token.